examples/vaex/my_script.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import sys
+
+from hamilton import base, driver
+from hamilton.plugins import h_vaex, vaex_extensions  # noqa F401
```

Comment:
@tyapochkin we can drop `vaex_extensions` here and instead add `vaex` to https://github.com/DAGWorks-Inc/hamilton/blob/main/hamilton/function_modifiers/base.py#L23